python/gui/auto_generated/qgshighlight.sip.in

```diff
@@ -18,6 +18,9 @@ for highlighting features or geometries on a map canvas.
 
 .. code-block:: python
 
+       mapCanvas = iface.mapCanvas()
```

Comment:
Is it possible to move this bit to a "setup" section?

Reply:
When running them, snippets comes directly from compiled Python API docstrings (.so), so i don't want to put setup section in the docstring, because it is readable by IDE and more.
I would prefer move this specific line could be moved to the python test file 'test_docstrings.py" and mapCanvas passed as a global like iface variable.